sdk/src/main/java/com/google/cloud/dataflow/sdk/io/FileBasedSource.java

```diff
@@ -120,10 +132,14 @@ public FileBasedSource(String fileName, long minBundleSize,
       long startOffset, long endOffset) {
```

Comment:
Noting that there is an API gap here -- you can't do a known offset with a VP filename.

This is probably okay, because if you don't know what file you're reading how could you know what positions you need to read?